service.go

```diff
@@ -423,16 +442,62 @@ func populateService(k8sService *k8score.Service, acks map[string]string) servic
 			log.WithError(err).Warnf("Cannot parse isDaemon label value for service with name %s.", serviceName)
 		}
 	}
+	appPort := getAppPortForService(k8sService)
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**issue:** ‏This is not working for the `notifications-push` deployments, it uses 8080 instead of the [configured port for the service which is 8599](https://github.com/Financial-Times/notifications-push/blob/master/helm/notifications-push/app-configs/notifications-push_eks_delivery.yaml#L17).

<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**suggestion:** ‏I think we should change the implementation of `getAppPortForService` to look for the value [here](https://github.com/Financial-Times/notifications-push/blob/master/helm/notifications-push/templates/service.yaml#L14) (at least for the case of notifications-push).